core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala

```diff
@@ -423,18 +430,38 @@ private[spark] class MesosCoarseGrainedSchedulerBackend(
     tasks.toMap
   }
 
+  /** Extracts task needed resources from a list of available resources. */
+  private def partitionTaskResources(resources: JList[Resource], taskCPUs: Int, taskMemory: Int)
+    : (List[Resource], List[Resource]) = {
+
+    // partition cpus & mem
+    val (afterCPUResources, cpuResourcesToUse) =
+      partitionResources(resources, "cpus", taskCPUs)
+    val (afterMemResources, memResourcesToUse) =
+      partitionResources(afterCPUResources.asJava, "mem", taskMemory)
+
+    // partition ports
+    val (afterPortResources, portResourcesToUse) =
+    partitionPortResources(nonZeroPortValuesFromConfig(sc.conf), afterMemResources)
```

Comment:
You mean the vals or ":".? According to scala style guide return types should have two space indent.
https://github.com/databricks/scala-style-guide#indent